JobProcessor.App/Controllers/JobsController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using JobProcessor.App.ViewModels;
+using JobProcessor.Domain.Services;
+using System;
+using System.Web.Mvc;
+
+namespace JobProcessor.App.Controllers
+{
+    public class JobsController : Controller
+    {
+        protected readonly IJobService jobService;
+        public JobsController(IJobService jobService)
+        {
+            this.jobService = jobService;
+        }
+
+        public ActionResult Create(JobCreationViewModel job)
+        {
+            ModelState.Clear();
```

Comment:
You clear ModelState but you don't clear data, so every time I go to the Create form I see data that I put past time

Reply:
Also clearing ModelState is not needed if you clear data, meaning create a new object